### PR TITLE
Merge gz-sim8 ➡️  main

### PR DESCRIPTION
**Pull request by scpeters**
# ➡️  Forward port

Port `gz-sim8 ` ➡️  `main`

Branch comparison: https://github.com/ignitionrobotics/ign-gazebo/compare/main...gz-sim8

**Note to maintainers**: Remember to **Merge** with commit (not squash-merge or rebase)

### CODE REVIEW

File: examples/plugin/custom_sensor_system/CMakeLists.txt [hunk 1]
```diff
@@ -20,7 +20,7 @@ include(FetchContent)
 FetchContent_Declare(
   sensors_clone
   GIT_REPOSITORY https://github.com/gazebosim/gz-sensors
-  GIT_TAG main
+  GIT_TAG gz-sensors8
```
[the remark this answers is not in the document]
Comment (scpeters) [reply examples/plugin/custom_sensor_system/CMakeLists.txt]: fixed with force push to https://github.com/gazebosim/gz-sim/commit/734977afcb1b2964e29dabd852eb4cfcf47e070f

**Comment by scpeters** [comment]
build failing; needs a forward port of https://github.com/gazebosim/gz-math/pull/551

**Comment by scpeters** [comment]
> build failing; needs a forward port of [gazebosim/gz-math#551](https://github.com/gazebosim/gz-math/pull/551)

https://github.com/gazebosim/gz-math/pull/570

**Comment by mjcarroll** [comment]
@osrf-jenkins retest this please

**Comment by scpeters** [comment]
I'll rebuild the gz-math nightly and then run CI again

**Comment by scpeters** [comment]
with the new nightly things are better; CI is as good as it's going to get without disabling tests

**Review by iche033 — APPROVED** [review]

**Pull request merged**